docs/CONFIGURATION-EXAMPLES.md

```diff
@@ -0,0 +1,142 @@
+<!---
+id: configuration
+title: Configuring the Testrunner Toolkit
+sidebar_label: Configuration
+--->
+# Configuring the Testrunner Toolkit
+The Testrunner Toolkit (`saucectl`) requires a configuration file to know which tests to run along with which framework to use.
+
+## Basic Configuration
+
+By default, `config.yml` is the file `saucectl` looks to for its configuration.
+
+__Simple `config.yml` using `puppeteer`__
+
+```yaml
+apiVersion: v1
+metadata:
+  name: Feature XYZ
+  tags:
+    - e2e
+    - release team
+    - other tag
```

Comment:
These are not supported yet. I kept them around to give a glance on how things could look like and I wonder whether or not we should continue doing that.

Reply:
No worries. I can remove those tags from the page, but it also links to the file so that will provide the most up-to-date context for readers.